frontend/app/src/scenes/Home/scenes/UpsertCategory/index.tsx

```diff
@@ -139,7 +203,92 @@ class UpsertCategory extends React.Component<
                       >
                         <Input placeholder="ex: KG & Primary" />
                       </Form.Item>
-                      <Form.Item>
+                      <Title level={3}>Translations</Title>
+                      {Array.from(this.state.translations.keys()).map(
+                        (language) => {
+                          const translation = this.state.translations.get(
+                            language
+                          );
+                          const translationMeta = translationMetaData.get(
+                            language
+                          );
+                          if (translationMeta && translation) {
+                            return (
+                              <Card
+                                key={language}
+                                className={mainStyles.cardMargin}
+                              >
+                                <Button
+                                  type="link"
+                                  className={mainStyles.closeButton}
+                                  onClick={() => this.handleClose(language)}
+                                >
+                                  <DeleteOutlined />
+                                </Button>
+                                <Title level={3}>{translationMeta.name}</Title>
+                                <Text>{translationMeta.title}</Text>
+                                <Input
+                                  className={mainStyles.formItem}
+                                  value={translation.name}
+                                  placeholder={translationMeta.placeholder}
+                                  onChange={(event) =>
+                                    this.handleTranslations(
+                                      language,
+                                      event.target.value
+                                    )
+                                  }
+                                />
+                              </Card>
+                            );
+                          }
+                        }
+                      )}
+                      {!this.state.isAddTranslationSelected && (
```

Comment:
change this variable name. 

ex: something like `isInLanguageSelectionMode`

Reply:
or `isLanguageSelectionModeEnabled`

Reply:
okay